lib/widgets/large_screen.dart

```diff
@@ -13,7 +13,7 @@ class LargeScreen extends StatelessWidget {
         Expanded(
             flex: 5,
             child: Container(
-                padding: EdgeInsets.symmetric(horizontal: 16, vertical: 16),
+                // padding: EdgeInsets.symmetric(horizontal: 8, vertical: 8),
```

Comment:
You do not need to comment out code. b/c the repo is using git just commit before you make this change and delete the text. If it happens that you made a mistake
``` git
git revert --hard
```
will revert the code to what it was when you last commited.